examples/pylab_examples/vline_hline_demo.py

```diff
@@ -7,26 +7,18 @@
 """
 
 import matplotlib.pyplot as plt
-from matplotlib.transforms import blended_transform_factory as btf
 import numpy as np
 
 
-def f(t):
-    s1 = np.sin(2 * np.pi * t)
-    e1 = np.exp(-t)
-    return np.abs(s1 * e1) + .05
-
 t = np.arange(0.0, 5.0, 0.1)
-s = f(t)
+s = np.exp(-t) + np.sin(2 * np.pi * t) + 1
 nse = np.random.normal(0.0, 0.3, t.shape) * s
 
-fig = plt.figure(figsize=(12, 6))
-vax = fig.add_subplot(121)
-hax = fig.add_subplot(122)
+fig, (vax, hax) = plt.subplots(1, 2, figsize=(12, 6))
 
 vax.plot(t, s + nse, '^')
 vax.vlines(t, [0], s)
-vax.vlines([1, 2], 0, 1, transform=btf(vax.transData, vax.transAxes), colors='r')
+vax.vlines([1, 2], 0, 1, transform=vax.get_xaxis_transform(), colors='r')
```

Comment:
I feel like `vax.get_xaxis_transform()` would benefit from an explanatory comment of why it's in this code.